src/main/resources/static/codegen/built-cas2-api-spec.yml

```diff
@@ -241,11 +241,11 @@ paths:
     get:
       tags:
         - People operations
-      summary: Searches for a Person by their CRN
+      summary: Searches for a Person by their Prison Number (NOMIS ID)
```

Comment:
Would it be clearer if this were part of the same commit where `cas2-api.yml` is modified?